packages/twenty-front/src/pages/settings/data-model/SettingsObjectOverview/EdgeUtil.ts

```diff
@@ -0,0 +1,60 @@
+export const calculateSourcePosition = (
```

Comment:
All these components should go to the `modules` section and note the `pages` one.
We intend to keep the page as simple as possible. Ideally, pages should only be an empty container calling a component from modules. 

modules folder is organized in components/hooks/types/utils... and we don't want to replicate this structure in pages, that's why pages should only contain entrypoints.